src/main/scala/org/ergoplatform/explorer/db/dao/OutputsOps.scala

```diff
@@ -62,6 +62,12 @@ object OutputsOps extends DaoOps with JsonMeta {
     fr"LEFT JOIN node_transactions t ON o.tx_id = t.id LEFT JOIN node_headers h ON h.id = t.header_id" ++
     fr"WHERE o.ergo_tree = $ergoTree").query[ExtendedOutput]
 
+  def findByErgoTreeRoot(ergoTreeRoot: String): Query0[ExtendedOutput] =
+    (fr"SELECT" ++ allFieldsRefFr("o") ++ fr", i.tx_id, h.main_chain" ++
+      fr"FROM node_outputs o LEFT JOIN node_inputs i ON o.box_id = i.box_id" ++
+      fr"LEFT JOIN node_transactions t ON o.tx_id = t.id LEFT JOIN node_headers h ON h.id = t.header_id" ++
+      fr"WHERE o.ergo_tree LIKE ${"%" + ergoTreeRoot}").query[ExtendedOutput]
```

Comment:
Please describe in ScalaDoc: 1) the logic 2) which query was used as a prototype 3) assumptions about ergo_tree content. 4) refer to original descriptions of segregated constants.

Reply:
Done.